www/src/views/showcase/featured-sites.js

```diff
@@ -58,244 +47,219 @@ const GradientOverlay = () => {
-                      "&:hover": { ...screenshotHover },
-                    },
-                  }}
-                  to={node.fields.slug}
-                  state={{ isModal: true }}
-                >
-                  {node.childScreenshot && (
-                    <Img
-                      fluid={
-                        node.childScreenshot.screenshotFile.childImageSharp
-                          .fluid
-                      }
-                      alt={node.title}
-                      sx={{ ...screenshot }}
-                    />
-                  )}
-                  <div>
-                    <span className="title">{node.title}</span>
-                  </div>
-                </Link>
-                <div
-                  sx={{
-                    color: `textMuted`,
-                    fontSize: 1,
-                    fontWeight: `body`,
-                    [mediaQueries.lg]: {
-                      marginTop: `auto`,
-                    },
-                  }}
-                >
-                  {node.built_by && (
-                    <div
-                      sx={{ color: `text`, fontFamily: `heading`, fontSize: 2 }}
-                    >
-                      Built by {node.built_by}
-                    </div>
-                  )}
-                  <ShowcaseItemCategories
-                    categories={node.categories}
-                    onCategoryClick={c => setFilters(c)}
+                {node.childScreenshot && (
+                  <Img
+                    fluid={
+                      node.childScreenshot.screenshotFile.childImageSharp.fluid
+                    }
+                    alt={node.title}
+                    sx={{ ...screenshot }}
                   />
+                )}
+                <div>
+                  <span className="title">{node.title}</span>
                 </div>
+              </Link>
+              <div
+                sx={{
+                  color: `textMuted`,
+                  fontSize: 1,
+                  fontWeight: `body`,
+                  mt: [null, null, null, `auto`],
+                }}
+              >
+                {node.built_by && (
+                  <div
+                    sx={{ color: `text`, fontFamily: `heading`, fontSize: 2 }}
+                  >
+                    Built by {node.built_by}
+                  </div>
+                )}
+                <ShowcaseItemCategories
+                  categories={node.categories}
+                  onCategoryClick={c => setFilters(c)}
+                />
               </div>
-            ))}
-            <div sx={{ display: `flex` }}>
-              <a
-                href="#showcase"
+            </div>
+          ))}
+          <div sx={{ display: `flex` }}>
+            <a
+              href="#showcase"
+              sx={{
+                backgroundColor: `card.background`,
+                borderRadius: 1,
+                marginRight: t => `${t.space[6]} !important`,
```

Comment:
```suggestion
                mr: t => `${t.space[6]} !important`,
```
can we also see if !important is necessary here, or can we remove it?